### PR TITLE
add check for managed clusters status for relocate modal

### PR DESCRIPTION
**Pull request by SanjalKatiyar**
https://bugzilla.redhat.com/show_bug.cgi?id=2134479
Screenshot from 2022-10-18 15-05-36

### CODE REVIEW

**Comment by SanjalKatiyar** [comment]
@GowthamShanmugam PTAL

File: packages/mco/components/disaster-recovery/app-failover-relocate/failover-relocate-modal-body.scss [hunk 1]
```diff
@@ -17,3 +17,8 @@
     width: 14rem;
   }
 }
+
+.mco-dr-modal--overflow {
```
Comment (GowthamShanmugam) [inline packages/mco/components/disaster-recovery/app-failover-relocate/failover-relocate-modal-body.scss:21]: Just a information:
   Removing ``hasNoBodyWrapper``` arg from Modal will takecare of the scrolling part. But when I expand the subscription group dropdown the toggle is hiding behind the footer.

Comment (SanjalKatiyar) [reply packages/mco/components/disaster-recovery/app-failover-relocate/failover-relocate-modal-body.scss:21]: ack...
this is how it is looking currently when opening subscription group dropdown:
Screenshot from 2022-10-18 18-12-08

File: packages/mco/components/disaster-recovery/app-failover-relocate/failover-relocate-modal-body.tsx [hunk 2]
```diff
@@ -36,14 +36,14 @@ import {
...
 import { TargetClusterSelector } from './target-cluster-selector';
-import './failover-relocate-modal-body.scss';
 
 export const findErrorMessage = (
   errorMessage: ErrorMessage,
   includeUpdateError: boolean = true
 ) =>
   [
     errorMessage.drPolicyControlStateErrorMessage,
+    errorMessage.managedClustersErrorMessage,
```
Comment (GowthamShanmugam) [inline packages/mco/components/disaster-recovery/app-failover-relocate/failover-relocate-modal-body.tsx:46]: i did a mistake here, We need to remove ```errorMessage.failoverAndRelocateActionErrorMessage``` from here, we need to keep only ```includeUpdateError && errorMessage.failoverAndRelocateActionErrorMessage```

Comment (SanjalKatiyar) [reply packages/mco/components/disaster-recovery/app-failover-relocate/failover-relocate-modal-body.tsx:47]: done

**Review by GowthamShanmugam — APPROVED** [review]

**Pull request merged**